syntax/std_os_nonwasm.go

```diff
@@ -21,12 +22,21 @@ func stdOsGetArgs() rel.Value {
 			// to handle running script from syntax library
 			return rel.NewArray()
 		} else {
-			offset = 2
+			offset = lookupCommandOffset()
 		}
 	}
 	return strArrToRelArr(os.Args[offset:])
 }
 
+func lookupCommandOffset() int {
+	for index, item := range os.Args[1:] {
+		if !strings.HasPrefix(item, "-") {
+			return index + 2
+		}
+	}
+	return 2
+}
```

Comment:
See if it's cleaner to get the `run` and `eval` commands to supply the arguments directly.

Reply:
Fixed.